profile.md

```diff
@@ -195,6 +195,23 @@ deployment:
   "organization_name": "system"
   ```
 
+- `organization_annotations`
+
+  Version: 2.15
+
+  The [annotations](http://v3-apidocs.cloudfoundry.org/version/3.69.0/index.html#metadata) attached to the organization that a Service Instance is associated with.
+  Note that the annotations of an organization in Cloud Foundry MAY be changed.
+  This OPTIONAL property holds an object with the annotations key/value pairs. If present, this property MUST be an object, with one or more properties as follows:
+  
+  ```
+  "organization_annotations": { "prefix-here.org/name-here":"value-here" }
```

Comment:
@gberche-orange In the OSBAPI call we’ve realised that CF won’t have the prefixes here.

Is there expectation that the annotation object will always need such prefixes?, or is it only for K8S example?

Reply:
@waterlink

I was hoping that the CF CAPI team would handle https://github.com/cloudfoundry/cloud_controller_ng/issues/1335 bringing planned Cf/K8S consistency and that prefixes could be used to filter out private annotation keys as mentionned in https://github.com/openservicebrokerapi/servicebroker/issues/654#issuecomment-485655814 in a consistent way between CF and K8S:

> As a 1st step, we can leverage K8S annotations naming conventions reproduced below to allow users to distinguish between public annotations (shared with all brokers) and private annotations (not shared with brokers):

I'm still waiting for CF CAPI response on this. The CF SAPI team could consider other alternative options for filtering private annotations though, and we could relax the CF examples in OSB API PR